src/08_example/01_running_async_code.md

```diff
@@ -0,0 +1,77 @@
+# Running Asynchronous Code
+An HTTP server should be able to serve multiple clients concurrently.
+Waiting for previous requests to complete before handling the current request can cause performance to suffer.
+The book [solves this problem](https://doc.rust-lang.org/book/ch20-02-multithreaded.html#turning-our-single-threaded-server-into-a-multithreaded-server) by creating a thread pool where each connection is handled on its own thread.
```

Comment:
Some of these lines exceed the github limit (80 chars?). Perhaps just check if there's an existing limit in the repo and format all lines accordingly? (There's a line break formatting for markdown in VSCode in case you're using that.)

Reply:
Looks like the github limit is 125 chars, so I wrapped everything (other than the links which are too long) to be less than that. I didn't see any max line length for this repo.